packages/shared/src/components/Feed.tsx

```diff
@@ -198,15 +203,17 @@ export default function Feed<T>({
 
   const showScrollOnboardingVersion =
     sidebarRendered &&
+    feedName === 'popular' &&
+    !isLoading &&
     alerts?.filter &&
+    !user?.id &&
     Object.values(ScrollOnboardingVersion).includes(scrollOnboardingVersion);
-  const fetchPageFn = showScrollOnboardingVersion
-    ? () => setShowFeedFilters(true)
-    : fetchPage;
+  const shouldScrollBlock =
+    scrollOnboardingVersion !== ScrollOnboardingVersion.Control;
```

Comment:
What will this evaluate if it's `null` or `undefined`?
Just to make sure when we might clear it at one stage.

Reply:
`Control` is defiend as default in `src/lib/featureManagement.ts`, so only if we change that then this would not work. But since `Control` is here what `null` usually is for other tests I think its fine.

Reply:
Yep agreed, but we have to make sure if we ever remove the flag (because there is a winner maybe) what will happen.
I guess we can then also modify this, but maybe it's safer to do the evaluation the otherway around?

To check if [V1, V2].includes(scrollOnboardingVersion).

Just a next level of security for someone who might join later, and is not aware of it.

Reply:
Fine with me and it will help you sleep easier 😴 😆